bluesky/tests/test_examples.py

```diff
@@ -360,24 +359,24 @@ def sim_kill():
         os.kill(pid, signal.SIGINT)
 
     scan = [Msg('checkpoint'), Msg('wait_for', [ev.wait(), ]), ]
-    assert_equal(RE.state, 'idle')
+    assert RE.state == 'idle'
     start = ttime.time()
     loop.call_later(1, sim_kill)
     loop.call_later(2, done)
 
     RE(scan)
-    assert_equal(RE.state, 'paused')
+    assert RE.state == 'paused'
     mid = ttime.time()
     RE.abort()
-    assert_equal(RE.state, 'idle')
+    assert RE.state == 'idle'
     stop = ttime.time()
 
     assert mid - start > 1
     assert stop - start < 2
 
-
 def test_abort():
-    raise KnownFailureTest
+    pytest.xfail("This needs a better reason why we know it fails."
```

Comment:
This is for some reason not successful on OSX. But the issue is not reproducible interactively. I concluded it was an issue with nose. Perhaps py.test fares better?